roles/libvirt/tasks/tftp.yml

```
@@ -33,10 +33,15 @@
   shell: cp /usr/share/syslinux/{pxelinux.0,menu.c32,chain.c32} /var/lib/tftpboot
 
 - name: 'chown /var/lib/tftpboot for foreman-proxy user'
-  shell: chown -R foreman-proxy:nobody /var/lib/tftpboot
+  file:
+    path: /var/lib/tftpboot
+    state: directory
+    owner: foreman-proxy
+    group: nobody
+    recursive: true
```

```suggestion
    recurse: true
```

at least if I trust https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-recurse

---

I think I read the description and got recursively stuck in my head.